server/src/test/java/org/elasticsearch/cluster/coordination/PersistedStateIT.java

```diff
@@ -54,4 +49,13 @@ public void testPersistentSettingsOnFullRestart() throws Exception {
                         .get(MetaData.SETTING_CLUSTER_MAX_SHARDS_PER_NODE.getKey()), equalTo(Integer.toString(maxShardsPerNode)));
 
     }
+
+    public void testIndexIsPreservedOnFullClusterRestart() throws Exception {
```

Comment:
Aren't there already other tests covering this?
Can we remove this class now and instead activate the missing Zen2 tests?